lib/foreman/renderer/scope/macros/base.rb

```diff
@@ -334,6 +334,22 @@ def gem_version_compare(first, second)
             Gem::Version.new(first.to_s) <=> Gem::Version.new(second.to_s)
           end
 
+          apipie :method, "Returns content of 'SSL CA file' configured in Settings > Authentication" do
+            example "SSL_CA_CERT=$(mktemp)
+                     cat > $SSL_CA_CERT <<CA_CONTENT
+                     <%= foreman_server_ca_cert %>
+                     CA_CONTENT
+                     curl --cacert $SSL_CA_CERT https://smart-proxy.example.com:8443"
+          end
+          def foreman_server_ca_cert
```

Comment:
I still think this doesn't really match the description.

Reply:
FTR being addressed at https://github.com/theforeman/foreman/pull/8074